### DISCUSSION TITLE
Tankstellen

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#198] [Q&A] [answered]
Wo kann man einstellen das Tankstellen in der Karte angezeigt werden ?
Momentan werden mir weder in den offline als auch in den online Karten keine Tankstellen angezeigt.

**Comment by devemux86** [comment, marked as answer]
@1maxi2 please follow these steps:

- Download offline Mapsforge maps
- Select map theme "Biker" or "MotoRider"
- Select the map layers with map menu

@1maxi2 folgen Sie bitte diesen Schritten:

- Offline Mapsforge-Karten herunterladen
- Wählen Sie das Kartenthema "Biker" oder "MotoRider"
- Wählen Sie die Kartenebenen mit dem Kartenmenü

<img src="https://github.com/devemux86/cruiser/assets/3484020/d5271608-1392-4d1d-a46e-a37d0759006d" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/b7a70735-bc69-42c9-97be-b341d6fc1730" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/6f7a82fe-d32b-47f1-bec7-7d6eb76fbdba" width="33%"/>

**Reply by 1maxi2** [reply]
Danke !

**Reply by 1maxi2** [reply]
Kann man das vielleicht so programmieren das Tankstellen standardmäßig immer angezeigt werden ( in " Biker ") Da Tankstellen ja wohl jeder Biker benötigt 🙂

**Reply by devemux86** [reply]
> Kann man das vielleicht so programmieren das Tankstellen standardmäßig immer angezeigt werden ( in " Biker ") Da Tankstellen ja wohl jeder Biker benötigt :slightly_smiling_face:

This should be possible.
Das sollte möglich sein.

**Reply by 1maxi2** [reply]
Wenn man zwischendurch auf online Karten wechselt und dann wieder auf offline zurückwechselt ist standardmäßig " Vegetation " ausgewählt.

**Reply by devemux86** [reply]
Currently "Vegetation" is the default.
We can also activate "Fuel stations" and any other map layer.

Derzeit ist "Vegetation" die Standardeinstellung.
Wir können auch "Tankstellen" und jede andere Kartenebene aktivieren.

**Reply by 1maxi2** [reply]
Ich würde sagen Tankstelle ist das wichtigste. Alles andere kann man ja nach den eigenen Bedürfnissen einstellen.

**Reply by WolfgangZ42** [reply]
Nein - Tankstellen benötige ich nur in ganz seltenen Ausnahmefällen, wenn z. B. irgendein Dussel wieder mal vergessen hat, mit vollgetanktem Mopped zum Start zu erscheinen. Ich möchte nicht ein solches Bild auf meinem Smartphone:
<img width="400" src="https://github.com/devemux86/cruiser/assets/121806238/822dbd2d-18d9-45e9-bef0-364eb6392730"/>
No - I only need gas stations in very rare exceptional cases, for example, when some fool has once again forgotten to show up at the start with a full tank of gas. I do not want such a picture on my smartphone.

**Reply by 1maxi2** [reply]
O.k. Das ist auch ein Argument. Dann belassen wir die Einstellung wie sie jetzt ist.

**Reply by SF4User** [reply]
Today I tried to set the display of POIs - e.g. gas stations. . . after 3-4 hours I gave up!
I've read this forum up and down, but all the explanations above get nowhere (for me).
My Cruiser is 3.022 (new update) an I performed exact the explanations in the description before.
After Download of the offline Mapsforge Maps the second and third Chard to select map theme "Biker" or "MotoRider" was nowhere to find. I even uninstall and completely reinstall the Cruiser App and deleted all Maps and reloaded all Mapsforge Maps new . . .
So . . . my suggestion: In Kurviger #2 there was a simple setting under: [ Adjustments ] . . . [ Maps ] . . . [ Levels ] then a menu in which you could find an icon for each POI and then switch it to "blue-on" - or "gray-off" so, petrol stations and/or restaurants and/or workshops etc. to on/off
That was very easy, independent of used Map's or Routing's and you could also do it on the go if you needed a gas station quickly, for example. After that was simply switched it off again so that the card didn't get too full.
I have a' Dream . . . For me! this feature realized and cruiser is perfect - no more wishes!

**Reply by WolfgangZ42** [reply]
It's strange that you didn't find it - it's quite the same as in Kurviger2:

<img width="150" src="https://github.com/devemux86/cruiser/assets/121806238/b083039e-9af5-468a-9cab-84f6e2515c27"/>   <img width="150" src="https://github.com/devemux86/cruiser/assets/121806238/a1d788aa-ba1c-4610-b0db-ddcae12c2138"/>    <img width="150" src="https://github.com/devemux86/cruiser/assets/121806238/520fbf92-37d2-428a-9499-9fc06538dc4e"/>   <img width="150" src="https://github.com/devemux86/cruiser/assets/121806238/e2cb317d-c139-465b-99ed-a7a1cec8f79a"/>   <img width="150" src="https://github.com/devemux86/cruiser/assets/121806238/daae1fde-6258-4c21-87d2-88dbe617585b"/>   

So with 4 taps you can switch the display of fuel stations on (and off!) - therefore there is no default setting necessary.

**Reply by devemux86** [reply]
@SF4User 

Map layers can be turned on / off with offline maps.
Have you downloaded and activated offline maps?
(they are automatically activated on download)

Then the menu "Map | Map layers" appears to click it.

Please see the images by @WolfgangZ42 and @DolomitesTom:
- https://github.com/devemux86/cruiser/discussions/198#discussioncomment-6067389
- https://github.com/devemux86/cruiser/discussions/198#discussioncomment-6067308

Did you find it? Do you need more help?

**Reply by SF4User** [reply]
That would be perfect! but, my problem seems to be that I can't see the map layer menu to my display.
See here my Screenshots:
![Screenshot Cruiser](https://github.com/devemux86/cruiser/assets/132825580/c9dfbd46-553e-415e-a859-b84eb744583c)

**Reply by devemux86** [reply]
@SF4User what have you selected in "Settings | Map | Advanced | Rendering engine"?
The default and recommended is "VTM (OpenGL)".

Then you can select the default "Biker" (or "MotoRider) map theme.
And the map layers menu will be available.

Your images show that you use a map theme without map layers?

---

Also select the default "Settings | Application | Advanced | Menu | Normal",
so the menu return to their default normal appearance without extra buttons.
(if you do not open external maps / themes)

**Reply by SF4User** [reply]
(...) "Then you can select the default "Biker" (or "MotoRider) map theme. And the map layers menu will be available" . . .
YES ! that was the Problem . . . thank You for Service, perfect - its work now 👍😉

**Reply by SF4User** [reply]
I "played" with this function right now. . .
My recommendation: move the [ Map Layer ] Bottom one panel further forward - right after you've pressed the ( Burger ) on the main page.
Reason: On the way, the many POI's on the map are annoying - you only switch on the POI's for example, if you're suddenly standing there outside and ned  one important POI e.g. Gas Station. Otherwise it is switched off during active navigation.
This is more likely to be the case at gas stations/workshops. . . Sights etc. would be left on all the time.
Anyway - quick access would be better here.

**Comment by DolomitesTom** [comment]
The menu is perfect as it is. It was development with ideas of many users who are now used to it. I guess no one of the longer term users wants this be changed again. 
You download and activate the maps that you want, then in the menu Karten-Ebenen you select the details that you want, that's it. In settings you can search for every single setting in your language, for example "zoom" or "größe". You can change the zoom level where these symbols should be visible, you can adapt the size of them. You can do everything that you can't do with other apps, even not with K2. There were huge developments in the recent past.

<img src="https://github.com/devemux86/cruiser/assets/121371629/c47d0316-37b4-496a-a8c4-f7b0b8980d11" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/121371629/2c21d12a-6d72-4dee-91ba-68862df58f3b" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/121371629/bbbee041-27b9-424d-ad3e-96a03fc52705" width="33%"/>

**Reply by SF4User** [reply]
Thank You. That would be perfect! but, my problem seems to be that I can't see the map layer [ Karten-Ebenen ] menu to my display.
I'm use the same Maps (Mapsforge), but the Bottom: Karten-Ebene is not there in my Dialog Box m  [ Karte ]. All Maps are activated.
See here my Screenshots:
![Screenshot Cruiser](https://github.com/devemux86/cruiser/assets/132825580/26d26f17-d9e1-4f75-a835-02a68d27db46)

**Reply by devemux86** [reply]
@SF4User please see my answer above:
- https://github.com/devemux86/cruiser/discussions/198#discussioncomment-6069384

**Reply by SF4User** [reply]
Problem gelöst ! 👍😉

**Comment by 0709wiwiwi** [comment]
- And once everything is set to your liking be able to save this as a Preset. 
_According to everyone's **personal preference** with or without Petrol Station display._

https://github.com/devemux86/cruiser/discussions/165#discussioncomment-5901201

- Or hikers who by (personal) default prefer to display public water sources, for example.

Note.
_Map preferences & map elements are currently unsupported via .pref files.
Other key settings such as vehicle speed and announcement distances automatically correctly set._
https://github.com/devemux86/cruiser/discussions/193#discussioncomment-6056986

**Reply by devemux86** [reply]
> Map preferences & map elements are currently unsupported via .pref files.

Indeed, we could include them as well.

**Reply by 1maxi2** [reply]
[0709wiwiwi](https://github.com/0709wiwiwi)
[Vor 44 Minuten](https://github.com/devemux86/cruiser/discussions/198#discussioncomment-6068401)
Und sobald alles nach Ihren Wünschen eingestellt ist, können Sie es als Voreinstellung speichern.
Ganz nach persönlichem Geschmack mit oder ohne Tankstellenanzeige.

Wie bitte funktioniert das Speichern der persönlichen Einstellungen ?
z.B. " Tankstellen "

**Reply by 0709wiwiwi** [reply]
@1maxi2 
Most **but not all** current Cruiser settings can be exported and re-imported to re-create the current Cruiser experience at a later date.

This can be done through the following Cruiser function: 
Cruiser > Menu > Settings > Application > Advanced > Preferences  - Export & Import by a  .pref file to save.

**Not yet supported** are _Maps (Mapsforge) & Map Theme & Map Layers._

The preferences function call could be more direct by a comfortable Large Presets Button.
Find idea here:  [Application Presets](https://github.com/devemux86/cruiser/discussions/165#discussioncomment-5901201)

**Note**. 
_Cruiser gps is in fast development, not all proposals are implemented or even ever will and of course if technically possible. 
Anyway you can add a thumbs up for those suggestions or ideas you particularly like to be implemented in a future update._

**Reply by devemux86** [reply]
> Most but not all current Cruiser settings can be exported and re-imported to re-create the current Cruiser experience at a later date.

**All settings** that exist in the multiple "Settings" screens are exported.

> Not yet supported are Maps (Mapsforge) & Map Theme & Map Layers.

The active maps / themes are not currently exported, because users may have not their downloads when restore settings.
I need to think more about it.

**Reply by devemux86** [reply]
> Cruiser is in fast development

I would say that Cruiser is on a frenetic pace of development!
It's amazing how much has been implemented / changed in the last few months.

And how much work lies ahead. :slightly_smiling_face:

**Reply by 0709wiwiwi** [reply]
> I need to think more about it.

Supporting all possible vector maps and the themes 100% is immense, so it's better to limit the task and keep in control, right? 
Since you manage the Mapsforge maps yourself, should you so only support the corresponding **Mapsforge map themes and map layers.** And by default at least minimaly than provide and keep the Mapsforge World map activated into the Cruiser app.
[Mapsforge World map](https://github.com/devemux86/cruiser/discussions/101#discussioncomment-5438583url)

_The current Mapsforge map(s) should not be part of preservation and restoration using Presets.  
As if you prefer to show Petrol stations such applies for each ever choosen new Mapsforge map._
